packages/example-next/components/connectorCards/WalletConnectCard.tsx

```diff
@@ -26,8 +26,9 @@ export default function WalletConnectCard() {
 
   // attempt to connect eagerly on mount
   useEffect(() => {
-    walletConnect.connectEagerly().catch(() => {
+    walletConnect.connectEagerly().catch((e) => {
       console.debug('Failed to connect eagerly to walletconnect')
```

Comment:
nit: Maybe name this `error` for clarity? Also, can you include it in the console.debug as a second argument, or else use `console.error`?

Reply:
Done